pagopa/io-app
# fix(Bonus Pagamenti Digitali): [#176477833] Hidden CTA until markdown has finished loading

## Short description
In `BpdTransactionSummaryComponent.tsx`, we need to show the CTA after the bottom sheet's markdown content has finished loading.

## How to test 
1. Go to Wallet -> Cashback -> n. of transactions -> Anything Missing?
2. See the CTA being shown after MD has loaded
